node/resolvers/Routes/index.ts

```diff
@@ -161,11 +161,15 @@ export default {
             }
           )
           .then((res: any) => {
+            const { addresses, paymentTerms, customFields } =
+              res?.data?.getCostCenterById
```

Comment:
The code was doing a null-safe access at this point before, with `getCostCenterById?.addresses`, for example. Here we're assuming that `getCostCenterById` will always be an object if `res.data` is defined. Is that safe? Otherwise this should be:

```suggestion
              res?.data?.getCostCenterById ?? {}
```